src/firebase/auth/functions/index.js

```diff
@@ -22,34 +24,78 @@ async function sendGoodByeMessage(_user) {
 /**
  * @param {any} user The new user
  */
-async function sendWelcomeMessage(user) {
+// async function sendWelcomeMessage(user) {
```

Comment:
I commented out this other function for welcome message

Reply:
Alright @BenjaminSsempala , thanks!